docs/README.md

```diff
@@ -65,43 +51,20 @@ The Shelf is a theme block responsible for showing a **collection of products**
       "flex-layout.row#main",
       "shelf.relatedProducts"
     ]
-  },
+  }
+}
 ```
 
-### Shelf 
+| Prop name              | Type                                   | Description                                                                                                                                                                                                                               | Default value |
+| ---------------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `category`             | `String`                               | Category ID of the listed items. For sub-categories, use "/" (e.g. "1/2/3")                                                                                                                                                               | -             |
+| `specificationFilters` | `Array({ id: String, value: String })` | Specification Filters of the listed items.                                                                                                                                                                                                | []            |
+| `collection`           | `String`                               | Filter by collection.                                                                                                                                                                                                                     | -             |
+| `orderBy`              | `Enum`                                 | Ordination type of the items. Possible values: `OrderByTopSaleDESC`, `OrderByReleaseDateDESC`, `OrderByBestDiscountDESC`, `OrderByPriceDESC`, `OrderByPriceASC`, `OrderByNameASC`, `OrderByNameDESC` | `OrderByTopSaleDESC`          |
+| `hideUnavailableItems` | `Boolean`                              | Hides items that are unavailable.                                                                                                                                                                                                         | `false`       |
+| `maxItems` | `Number`                              | Maximum items to be fetched.                                                                                                                                                                                                         | `10`       |
 
-| Prop name                  | Type                             | Description                                                                                                                                                                                                                                            | Default value     |
```

Comment:
I would create a topic for the `specificationFilters` array here, below the props table. For example: 

- **`HiddenFacets` array**

| Prop name              | Type                   | Description                 | Default value |
| ---------------------- | ---------------------- | --------------------------- | ------------- |
| `Id`            | `String`      | Specification Filters ID     | `undefined`         |
| `value`           | `String`       | Specification Filters values  | `undefined`         |